.gitignore

```diff
@@ -24,4 +24,7 @@ test/background.js
 test/bundle.js
 test/test-bundle.js
 
-notes.txt
\ No newline at end of file
+notes.txt
+
+.coveralls.yml
```

Comment:
yes, i've added the repo token to the Ci build as an env_variable.  .coveralls.yml will be needed to run coveralls locally